server/http_hooks.go

```diff
@@ -286,3 +288,57 @@ func (p *Plugin) writeSubmitDialogResponse(w http.ResponseWriter, response *mode
 		p.API.LogError("failed to write DialogResponse", "err", err.Error())
 	}
 }
+
+func (p *Plugin) handleDynamicArgTest(w http.ResponseWriter, r *http.Request) {
+	queryArgs := []string{"user_input", "parsed", "root_id", "parent_id", "user_id", "site_url", "request_id", "session_id", "ip_address", "accept_language", "user_agent"}
+	query := r.URL.Query()
+
+	channelID := query.Get("channel_id")
+	teamID := query.Get("team_id")
+	userID := query.Get("user_id")
+	rootID := query.Get("root_id")
+
+	channel, _ := p.API.GetChannel(channelID)
+	team, _ := p.API.GetTeam(teamID)
+	user, _ := p.API.GetUser(userID)
+
+	argMap := map[string]string{}
+	for _, arg := range queryArgs {
+		argMap[arg] = query.Get(arg)
+	}
+	result := fmt.Sprintf("dynamic argument was triggered by **%v** from team **%v** in the **%v** channel, with these arguments __%v__", user.GetFullName(), team.DisplayName, channel.DisplayName, argMap)
```

Comment:
`argMap` looks a bit ugly: 
![Screenshot from 2020-08-05 13-25-39](https://user-images.githubusercontent.com/16541325/89407485-31578a00-d71f-11ea-9c67-ffe4153c096d.png)

What do you think of posting each element in a new line?